src/components/LeafletMap.svelte

```diff
@@ -0,0 +1,126 @@
+<script>
+  import deepEqual from 'deep-equal';
+  import throttle from 'lodash.throttle';
+  import { createEventDispatcher, onMount, onDestroy } from 'svelte';
+  import * as L from 'leaflet';
+  import 'leaflet/dist/leaflet.css';
+
+  export let index;
+  export let id;
+  export let center;
+  export let zoom;
+  export let mapStyle;
+  export let numberOfMaps;
+
+  const dispatch = createEventDispatcher();
+
+  let style = {};
```

Comment:
We aren't using `style` anywhere in here so I'm inclined to remove the variable, but if there's a reason to keep for consistency or something, that's fine too!